src/common/post.ts

```diff
@@ -1,14 +1,28 @@
-import { DataSource } from 'typeorm';
+import { DataSource, EntityManager } from 'typeorm';
 import fetch from 'node-fetch';
-import { User } from '../entity';
-import { Comment, ExternalLinkPreview, Post } from '../entity';
+import {
+  Comment,
+  ExternalLinkPreview,
+  FreeformPost,
+  Post,
+  PostOrigin,
+  SquadSource,
+  User,
+  WelcomePost,
+} from '../entity';
 import { ValidationError } from 'apollo-server-errors';
 import { isValidHttpUrl } from './links';
+import { markdown } from './markdown';
+import { generateShortId } from '../ids';
+
+const defaultWelcomeBanner =
+  'https://daily-now-res.cloudinary.com/image/upload/f_auto,q_auto/public/welcome_post';
 
 export const defaultImage = {
   urls: process.env.DEFAULT_IMAGE_URL?.split?.(',') ?? [],
   ratio: parseFloat(process.env.DEFAULT_IMAGE_RATIO),
   placeholder: process.env.DEFAULT_IMAGE_PLACEHOLDER,
+  welcomePost: process.env.WELCOME_POST_BANNER ?? defaultWelcomeBanner,
```

Comment:
There's no need to get it from env var and have it hard coded. Let's choose one :)

Reply:
Cool. Let me update that one.